translations/binaries/slps_strings_items_consumable.json

```
@@ -0,0 +1,2153 @@
+            "original_bytes_length": 75,
+            "available_bytes_size": 79
+        },
+        {
+            "offset": 1767912,
+            "string": "[Sign_Apple_Gel]パナシーアボトル[END]",
+            "translation": "[Sign_Apple_Gel]만병통치약[END]",
+            "original_bytes_length": 22,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1767880,
+            "string": "全ての状態異常を回復する薬。[LINE]\n[LINE]\n[END]",
+            "translation": "모든 상태이상을 회복하는 약.[LINE]\n[LINE]\n[END]",
+            "original_bytes_length": 31,
+            "available_bytes_size": 31
+        },
+        {
+            "offset": 1767856,
+            "string": "[Sign_Apple_Gel]リキュールボトル[END]",
+            "translation": "[Sign_Apple_Gel]리큐어 병[END]",
+            "original_bytes_length": 22,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1767792,
+            "string": "全ての状態異常・減少効果を[LINE]\n防止する薬。[LINE]\n戦闘中のみ有効。[END]",
+            "translation": "모든 상태이상・감소 효과를[LINE]\n방지하는 약.[LINE]\n전투 중에만 유효.[END]",
+            "original_bytes_length": 57,
+            "available_bytes_size": 63
+        },
+        {
+            "offset": 1767768,
+            "string": "[Sign_Apple_Gel]ライフボトル[END]",
+            "translation": "[Sign_Apple_Gel]생명의 병[END]",
+            "original_bytes_length": 18,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1767704,
+            "string": "戦闘不能を回復し、[LINE]\n最大ＨＰの４０％分[LINE]\nＨＰが回復する薬。[END]",
+            "translation": "전투 불능을 회복하고,[LINE]\n최대 HP의 40%만큼[LINE]\nHP가 회복되는 약.[END]",
+            "original_bytes_length": 57,
+            "available_bytes_size": 63
+        },
+        {
+            "offset": 1767680,
+            "string": "[Sign_Apple_Gel]エリクシール[END]",
+            "translation": "[Sign_Apple_Gel]엘릭서[END]",
+            "original_bytes_length": 18,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1767608,
+            "string": "戦闘不能を回復し、[LINE]\nＨＰを全快にする薬。[LINE]\n不老長寿の霊薬とも言われる。[END]",
+            "translation": "전투 불능을 회복하고,[LINE]\nHP를 전체 회복하는 약.[LINE]\n불로장수의 영약이라고도 한다.[END]",
+            "original_bytes_length": 69,
+            "available_bytes_size": 71
+        },
+        {
+            "offset": 1767584,
+            "string": "[Sign_Apple_Gel]オールディバイド[END]",
+            "translation": "[Sign_Apple_Gel]올 디바이드[END]",
+            "original_bytes_length": 22,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1767520,
+            "string": "敵味方が受けるダメージを半減する。[LINE]\nもう一度使うと元に戻る。[LINE]\n[END]",
+            "translation": "적아군이 받는 데미지를 반감한다.[LINE]\n다시 사용하면 원래대로 돌아간다.[LINE]\n[END]",
+            "original_bytes_length": 61,
+            "available_bytes_size": 63
+        },
+        {
+            "offset": 1767496,
+            "string": "[Sign_Apple_Gel]アワーグラス[END]",
+            "translation": "[Sign_Apple_Gel]모래시계[END]",
```

```suggestion
            "translation": "[Sign_Apple_Gel]시간의 병[END]",
```

ToD2에서 시간의 병으로 번역되었다고 함